src/script_functions.c

```diff
@@ -1349,9 +1340,9 @@ int c_get_window_frame_extents(lua_State *lua)
 		gulong *extents = 0;
 		int len = 0;
 
-		int ret = my_wnck_get_cardinal_list (wnck_window_get_xid(window),
-		                                     XInternAtom(dpy, "_NET_FRAME_EXTENTS", False),
-		                                     &extents, &len);
+		my_wnck_get_cardinal_list (wnck_window_get_xid(window),
+		                           XInternAtom(dpy, "_NET_FRAME_EXTENTS", False),
+		                           &extents, &len);
```

Comment:
With `-Wall` enabled in the makefile, this causes `warning: unused variable ‘ret’ [-Wunused-variable]`